### DISCUSSION TITLE
DeserializationExceptionHandler example

### DISCUSSION THREAD

**Discussion by dvargaslantana** [LGouellec/streamiz#288] [General]
Hello,

Are there any examples of the DeserializationExceptionHandler config being used? I would like to see a working example of how to implement something like this. Any help is appreciated!

Thanks

**Comment by LGouellec** [comment]
Hi @dvargaslantana, 

There is multiple examples in my unit test repository : https://github.com/LGouellec/kafka-streams-dotnet/blob/develop/test/Streamiz.Kafka.Net.Tests/Public/DeserializationExceptionHandlerTests.cs

Btw, you can easily implement your own handler, based on the type of the exception to CONTINUE or FAIL. 
In future, there will be a DLQ action to route the "wrong" message to a dead letter queue topic.

Best regards,

**Reply by dvargaslantana** [reply]
Perfect thank you!
